src/sagemaker/jumpstart/accessors.py

```diff
@@ -307,6 +299,21 @@ def get_model_specs(
                 model_specs.set_hub_content_type(HubContentType.MODEL_REFERENCE)
                 return model_specs
 
+            except Exception as ex:
+                logging.info(
+                    "Recieved exeption while calling APIs for ContentType ModelReference, \
```

Comment:
typo: `Recieved` - please also check error message with @judyheflin

Reply:
also, high-level question for future PRs: shall we retry on all types of error? As in, if retry throttling as well?

That may be fine, but just want to make sure that's a conscious decision.

Reply:
yeah I think its fine to retry on all error types but now that I am thinking about it I feel like I can just restrict retry with a different contentType only in the case of ResourceNotFound errors

Reply:
regardless engaging with @judyheflin for the error message